### PR TITLE
Add Front-End Functionality

### PR DESCRIPTION
**Pull request by taymoorKhalid**
React based frontEnd and Functionaity!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home/>} />
+      </Routes>
+    </BrowserRouter>
   );
 }
 
-export default App;
+export default App;
```
Comment (DanyalNazir10) [inline src/App.js:15]: Add EOF( End of file)

File: src/components/todoItem/TodoItem.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            ref={inputRef}
+          />
+          <button type="submit">Save</button>
+          <button type="button" onClick={handleCancelEdit}>
+            Cancel
+          </button>
+        </form>
+      ) : (
+        <>
+          <button className="item-left" onClick={toggleCompletion}>
```
Comment (DanyalNazir10) [inline src/components/todoItem/TodoItem.js]: Create common component for  buttons

File: src/components/todoItem/TodoItem.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+              fill={item.isCompleted ? "#22C55E" : "#0d0d0d"}
+            >
+              <circle
+                cx="11.998"
+                cy="11.998"
+                fill-rule="nonzero"
+                r="9.998"
+              ></circle>
+            </svg>
+            <p
```
Comment (DanyalNazir10) [inline src/components/todoItem/TodoItem.js]: No inline CSS.

File: src/components/todoItem/TodoItem.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+          </button>
+          <div class="item-right">
+            <button
+              onClick={() => {
+                setIsEditing(true);
+              }}
+            >
+              <span class="visually-hidden">Edit</span>
+              <svg
+                clip-rule="evenodd"
```
Comment (DanyalNazir10) [inline src/components/todoItem/TodoItem.js]: No inLine CSS

File: src/components/todoList/TodoList.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./todolist.css";
+import TodoItem from "../todoItem/TodoItem";
+
+const TodoList = (props) => {
```
Comment (DanyalNazir10) [inline src/components/todoList/TodoList.js]: Instead of using "props." directly destruct the properties.

**Review by DanyalNazir10 — CHANGES_REQUESTED** [review]

**Comment by taymoorKhalid** [comment]
1) Changed Index.js to index.js.
2) Created a separate svg file.

**Review by DanyalNazir10 — APPROVED** [review]

**Pull request merged**